### ISSUE TITLE
wondering why the loss just keeps on increasing

### ISSUE THREAD

**Issue by cloudyskyy** [mariob6/style_text#25]
At the end of epoch, the loss becomes 12, and the transferred sentences are hard to read, is it because the parameters?
The parameters are as follows: 
in_memory: True
max_len: 20
embedding_size: 100
dim_y: 200
dim_z: 500
batch_size: 32
epochs: 20
temperature: 0.1
lambda_GAN: 1.0
dropout: 0.5
max_loss: 1e10
grad_clip: 30
max_d_loss: 1.2
beam_width: 5
initial_noise: 0
noise_decay: 0.998

autoencoder {
  input_size: 100
  hidden_size: 700
  num_layers: 1
  dropout: 0.5
  learning_rate: 0.0005
  beta_0: 0.5
  beta_1: 0.999
  word_drop: 0.0
}

discriminator {
  in_channels: 1
  out_channels: 128
  kernel_sizes: 1
  kernel_sizes: 2
  kernel_sizes: 3
  kernel_sizes: 4
  kernel_sizes: 5
  embedding_size: 100
  dropout: 0.5
  learning_rate: 0.0001
  beta_0: 0.5
  beta_1: 0.999
  l_smoothing: 0.05
  l_flipping: 0.05
}
